### PR TITLE
Create 1.5.0-upgrade-guide.html.markdown

### PR DESCRIPTION
**Pull request by Zuhairahmed**
## Description

v1.5.0 Upgrade Guide 

Link to any related issue(s):

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update
- [x] Documentation fix/enhancement

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [x] I have read the Terraform contribution guidelines
- [ ] I have added tests that prove my fix is effective or that my feature works per HashiCorp requirements
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run make fmt and formatted my code

### CODE REVIEW

**Review by martinstibbe — APPROVED** [review]
LGTM

File: website/docs/guides/1.5.0-upgrade-guide.html.markdown [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+MongoDB Atlas Provider 1.5.0: Upgrade and Information Guide
+---
+
+# MongoDB Atlas Provider 1.5.0: Upgrade and Information Guide
+
+The Terraform MongoDB Atlas Provider version 1.5.0 has a number of new and exciting features and changes.
+
+New Features:
+* You can now manage AWS Serverless Private Endpoints [`mongodbatlas_serverless_private_endpoints`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/serverless_private_endpoints)
+* You can now manage AtlasGov Projects [`mongodbatlas_mongodbatlas_project`](https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs/resources/project)
```
Comment (themantissa) [inline website/docs/guides/1.5.0-upgrade-guide.html.markdown:15]: Can we also add a line to enable the provider for use with AtlasGov and point to the getting started guide which will have a new section?   The URL should be https://registry.terraform.io/providers/mongodb/mongodbatlas/latest/docs#configure-mongodb-atlas-for-government

**Review by themantissa — CHANGES_REQUESTED** [review]
One change to better highlight being able to enable the provider for AtlasGov.

File: website/docs/guides/1.5.0-upgrade-guide.html.markdown (hunk 1 above)
Comment (Zuhairahmed) [reply website/docs/guides/1.5.0-upgrade-guide.html.markdown:15]: Updated, also added link to AtlasGov considerations in Getting Started Guide. Let me know if all looks good

**Review by themantissa — APPROVED** [review]
LGTM!

**Pull request merged**